solidity/contracts/bridge/Bridge.sol

```diff
@@ -100,9 +113,26 @@ contract Bridge {
         bytes8 blindingFactor,
         bytes20 walletPubKeyHash,
         bytes20 refundPubKeyHash,
-        bytes4 refundLocktime
+        bytes4 refundLocktime,
+        address vault
     );
 
+    /// @notice Allows the Governance to mark the given vault address as trusted
+    ///         or no longer trusted. Vaults are not trusted by default.
+    ///         Trusted vault must meet the following criteria:
+    ///         - `IVault.onBalanceIncreased` must have a known, low gas cost.
+    ///         - `IVault.onBalanceIncreased` must never revert.
```

Comment:
`onBalanceIncreased` will be renamed to `receiveBalanceIncrease` in #110. Let's not forget to change this description?

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/110/commits/32892c8ebd63dc74b9c54c48da7894c0826f184c.